skimage/transform/tests/test_warps.py

```diff
@@ -355,6 +355,18 @@ def test_resize_dtype():
     assert resize(x_f32, (10, 10), preserve_range=True).dtype == x_f32.dtype
 
 
+def test_resize_anti_aliasing_clip():
```

Comment:
:+1:
I would suggest to add a test for uint8 data and we will be fine I think :wink: Thank you @abouysso

Reply:
Yes, also added a check with `preserve_range=True` and `preserve_range=False`. :smiley: :

Reply:
That's great @abouysso, you may save some lines of code by using the decorator `pytest.mark.parametrize` like for example [here](https://github.com/scikit-image/scikit-image/blob/master/skimage/registration/tests/test_ilk.py#L36) :wink: